specification/maps/data-plane/DataRegistry/stable/2022-12-01/dataregistry.json

```diff
@@ -0,0 +1,712 @@
+      "name": "subscription-key",
+      "in": "header"
+    },
+    "SasToken": {
+      "type": "apiKey",
+      "description": "This is a shared access signature token is created from the List SAS operation on the [Azure Maps resource](https://aka.ms/amauth) through the Azure management plane via Azure portal, PowerShell, CLI, Azure SDKs, or REST APIs.\n\n With this token, any application is authorized to access with Azure role-based access controls and fine-grain control to the expiration, rate, and region(s) of use for the particular token. In other words, the SAS Token can be used to allow applications to control access in a more secured way than the shared key.\n\n For publicly exposed applications, our recommendation is to configure a specific list of allowed origins on the [Map account resource](https://aka.ms/amauth) to limit rendering abuse and regularly renew the SAS Token.",
+      "name": "SAS Token",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "AADToken": [
+        "https://atlas.microsoft.com/.default"
+      ]
+    },
+    {
+      "AzureKey": []
+    },
+    {
+      "SasToken": []
+    }
+  ],
+  "responses": {},
+  "parameters": {
+    "ApiVersion": {
+      "name": "api-version",
+      "description": "Version number of Azure Maps API.",
+      "type": "string",
+      "in": "query",
+      "required": true,
+      "default": "2022-12-01",
+      "x-ms-parameter-location": "client"
+    },
+    "DataFormat": {
+      "name": "dataFormat",
+      "description": "Data format of the content being uploaded.",
+      "type": "string",
+      "in": "query",
+      "required": true,
+      "enum": [
+        "geojson",
+        "zip"
+      ],
+      "x-ms-enum": {
+        "name": "DataFormat",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "geojson",
+            "description": "[GeoJSON](https://tools.ietf.org/html/rfc7946) is a JSON based geospatial data interchange format."
+          },
+          {
+            "value": "zip",
+            "description": "Compressed data format."
+          }
+        ]
+      },
+      "x-ms-parameter-location": "method"
+    },
+    "UniqueDataId": {
+      "name": "udid",
+      "description": "The unique data id for the data registry. The `udid` must have been obtained from a successful [Data Upload API](https://docs.microsoft.com/en-us/rest/api/maps/data-v2/upload-preview) call.",
+      "type": "string",
+      "in": "path",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "DataDescription": {
+      "name": "description",
+      "description": "The description to be given to the data registry.",
```

Comment:
any length limitation for description?

Reply:
We don't have the limitation to description in our system. But it sounds reasonable to set the max length to be e.g. 1024 to prevent customers from sending too big payload.

Reply:
Set max length as 1024.